src/keyworddocs.jl

```diff
@@ -0,0 +1,46 @@
+  * `footerskip::Integer`: number of rows at the end of a file to skip parsing.  Do note that commented rows (see the `comment` keyword argument) *do not* count towards the row number provided for `footerskip`, they are completely ignored by the parser
+  * `transpose::Bool`: read a csv file "transposed", i.e. each column is parsed as a row
+  * `comment::String`: string that will cause rows that begin with it to be skipped while parsing. Note that if a row number header or `datarow` and `comment` are provided, the header/data row will be the first non-commented/non-empty row _after_ the row number, meaning if the provided row number is a commented row, the header/data row will actually be the next non-commented row.
+  * `ignoreemptyrows::Bool=true`: whether empty rows in a file should be ignored (if `false`, each column will be assigned `missing` for that empty row)
+  * `select`: an `AbstractVector` of `Integer`, `Symbol`, `String`, or `Bool`, or a "selector" function of the form `(i, name) -> keep::Bool`; only columns in the collection or for which the selector function returns `true` will be parsed and accessible in the resulting `CSV.File`. Invalid values in `select` are ignored.
+  * `drop`: inverse of `select`; an `AbstractVector` of `Integer`, `Symbol`, `String`, or `Bool`, or a "drop" function of the form `(i, name) -> drop::Bool`; columns in the collection or for which the drop function returns `true` will ignored in the resulting `CSV.File`. Invalid values in `drop` are ignored.
+  * `limit`: an `Integer` to indicate a limited number of rows to parse in a csv file; use in combination with `skipto` to read a specific, contiguous chunk within a file; note for large files when multiple threads are used for parsing, the `limit` argument may not result in an exact # of rows parsed; use `threaded=false` to ensure an exact limit if necessary
+  * `threaded::Bool`: [not applicable to `CSV.Rows`] whether parsing should utilize multiple threads; by default threads are used on large enough files (more than 5_000 cells), but isn't allowed when `transpose=true`; only available in Julia 1.3+
+  * `ntasks::Integer=Threads.nthreads()`: [not applicable to `CSV.Rows`] for multithreaded parsed files, this controls the number of tasks spawned to read a file in concurrent chunks; defaults to the # of threads Julia was started with (i.e. `JULIA_NUM_THREADS` environment variable or `julia -t N`)
```

Comment:
why do we need both `ntasks` and `treaded` kwargs?

Reply:
I guess we could potentially combine them and setting `ntasks=1` would essentially be the same as `threaded=false`.

Reply:
this is what I would assume.

Reply:
Ok, just pushed a commit that combines `threaded` and `ntasks`